tools/src/vendoring/config/expoGoConfig.ts

```diff
@@ -13,7 +13,30 @@ const config: VendoringTargetConfig = {
       source: 'https://github.com/aws-amplify/amplify-js.git',
     },
     'react-native-view-shot': {
-      source: 'https://github.com/gre/react-native-view-shot.git',
+      source: 'react-native-view-shot',
+      sourceType: 'npm',
+      excludeFiles: ['src/__tests__/**/*', 'windows/**/*', 'LICENSE', 'README.md'],
```

Comment:
```suggestion
      excludeFiles: ['src/__tests__/**/*', 'windows/**/*', 'LICENSE', 'README.md'],
```

i had README and LICENSE in webview and async-storage. we can align this

Reply:
Leave them there or remove them from the other modules?

Reply:
my original thought is just about to leave the local modules/ as similar as the package downloaded in node_modules/. not a strong opinion and open to get more feedback. which is your preference btw?

Reply:
I was just going to remove unnecessary files but I don't have a strong opinion on it either. We can leave them there. We won't be vendoring that many modules like this

Reply:
alright! let the files back. if that files change too much and pollute our git history. we can remove them later.